1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="./js/camilla-9000/app.js"></script>
```

Comment:
please put my javascript include back and add yours on the next line.

![https://media.giphy.com/media/CvpnjOcV9ZxsI/giphy-downsized.gif](https://media.giphy.com/media/CvpnjOcV9ZxsI/giphy-downsized.gif)